docs/known_issues/eessi-2023.06.md

```diff
@@ -26,3 +26,17 @@ export OMPI_MCA_pml='ucx'
 export OMPI_MCA_mtl='^ofi'
 ```
 </div>
+
+### `Bug in EESSI initialization and priority mechanisms: site OpenMPI or UCX not loaded`
```

Comment:
I don't think these are two seperate issues, right? Shouldn't we just add to the "Failed to modify UD QP to INIT... etc" that the OMPI_MCA_* environment variables can be done in `host_injections/...`? 

I was browsing the docs, I think we should have some general explaination of `host_injections` under `Advanced usage`. I.e. that this directory is used for site specific tuning. And then, in a paragraph there, we can add how it can be used to execute LMOD hooks for site specific tuning. We can give an example there of a hook (might as well use the example of setting these three OpenMPI environment variables). It's a good point, I wanted to make this documentation, but didn't get to it before my leave. Writing it as a todo for this afternoon... We can then refer to that from the "Failed to modify UD QP to INIT" known issue and give the code for the LMOD hook that can be used as a workaround for this particular issue.

Reply:
Docs PR for Lmod hooks made [here](https://github.com/EESSI/docs/pull/188). I'd simply put an example hook to resolve this issue under the existing `Failed to modify UD QP to INIT on mlx5_0: Operation not permitted` header, and reference the new docs on Lmod hooks for further information.

Reply:
@casparvl Adjusted on the known issues pages. Could you review it and if everything is okay, merge it?